src/components/Header/Header.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from 'react';
+import './Header.scss';
+import HomeIcon from '@material-ui/icons/Home';
+import SupervisorAccountIcon from '@material-ui/icons/SupervisorAccount';
+import NotificationsIcon from '@material-ui/icons/Notifications';
+import SearchIcon from '@material-ui/icons/Search';
+import LinkedInIcon from '@material-ui/icons/LinkedIn';
+import { NavigationIcon } from '../NavigationIcon/NavigationIcon';
+import { AuthContext } from '../../context/AuthProvider';
+
+export const Header: React.FC = () => {
+  const user = useContext(AuthContext);
+
+  return (
+    <header className="header">
+      <nav className="header__left">
```

Comment:
header__left is telling about location of the elements inside not what it is
it could be search

Reply:
it shouldn't be nav